ci-operator/config/openshift/openshift-tests-private/openshift-openshift-tests-private-release-4.14__amd64-nightly.yaml

```diff
@@ -510,7 +510,7 @@ tests:
       BASE_DOMAIN: qe.devcluster.openshift.com
       BASELINE_CAPABILITY_SET: None
       E2E_RUN_TAGS: '@amd64 and @aws-ipi and @disconnected and @network-ovnkubernetes
-        and not @fips and not @workloads'
+        and not @fips and not @workloads and not @storage'
```

Comment:
Seems we also need to modify the filter for 4.15 and upgrade jobs.
```
$ grep -nr 'as: aws-ipi-disconnected-sts-basecap-none-addcap-' ./ci-operator/config/*
./ci-operator/config/openshift/openshift-tests-private/openshift-openshift-tests-private-release-4.14__arm64-nightly.yaml:446:- as: aws-ipi-disconnected-sts-basecap-none-addcap-f28
./ci-operator/config/openshift/openshift-tests-private/openshift-openshift-tests-private-release-4.14__arm64-nightly.yaml:468:- as: aws-ipi-disconnected-sts-basecap-none-addcap-f28-destructive
./ci-operator/config/openshift/openshift-tests-private/openshift-openshift-tests-private-release-4.15__arm64-nightly.yaml:446:- as: aws-ipi-disconnected-sts-basecap-none-addcap-f28
./ci-operator/config/openshift/openshift-tests-private/openshift-openshift-tests-private-release-4.15__arm64-nightly.yaml:468:- as: aws-ipi-disconnected-sts-basecap-none-addcap-f28-destructive
./ci-operator/config/openshift/openshift-tests-private/openshift-openshift-tests-private-release-4.14__arm64-nightly-4.14-upgrade-from-stable-4.13.yaml:157:- as: aws-ipi-disconnected-sts-basecap-none-addcap-f28
./ci-operator/config/openshift/openshift-tests-private/openshift-openshift-tests-private-release-4.15__amd64-nightly-4.15-upgrade-from-stable-4.14.yaml:196:- as: aws-ipi-disconnected-sts-basecap-none-addcap-f28
./ci-operator/config/openshift/openshift-tests-private/openshift-openshift-tests-private-release-4.14__amd64-nightly-4.14-upgrade-from-stable-4.13.yaml:196:- as: aws-ipi-disconnected-sts-basecap-none-addcap-f28
./ci-operator/config/openshift/openshift-tests-private/openshift-openshift-tests-private-release-4.15__arm64-nightly-4.15-upgrade-from-stable-4.14.yaml:157:- as: aws-ipi-disconnected-sts-basecap-none-addcap-f28
./ci-operator/config/openshift/openshift-tests-private/openshift-openshift-tests-private-release-4.14__amd64-nightly.yaml:487:- as: aws-ipi-disconnected-sts-basecap-none-addcap-f28
./ci-operator/config/openshift/openshift-tests-private/openshift-openshift-tests-private-release-4.14__amd64-nightly.yaml:504:- as: aws-ipi-disconnected-sts-basecap-none-addcap-f28-destructive
./ci-operator/config/openshift/openshift-tests-private/openshift-openshift-tests-private-release-4.15__amd64-nightly.yaml:487:- as: aws-ipi-disconnected-sts-basecap-none-addcap-f28
./ci-operator/config/openshift/openshift-tests-private/openshift-openshift-tests-private-release-4.15__amd64-nightly.yaml:504:- as: aws-ipi-disconnected-sts-basecap-none-addcap-f28-destructive
```

Reply:
Good catch, added.